MediaBrowser.XbmcMetadata/Savers/BaseNfoSaver.cs

```diff
@@ -511,7 +515,7 @@ public static void AddMediaInfo<T>(T item, XmlWriter writer)
                 writer.WriteElementString("credits", person);
             }
 
-            foreach (var trailer in item.RemoteTrailers)
+            foreach (var trailer in item.RemoteTrailers.OrderBy(t => t.Url?.Trim()))
```

Comment:
This feels kinda hacky. Why do we need to trim strings coming from our DB?

Reply:
We could remove this here and do a upgrade/conversion pass... but a _lot_ of fields we have in the database have trailing blanks because the source INFO provider had blanks and we didn't trim when copying the data over. As for this _specific_ field, probably could remove because the `Uri`s are unlikely to be bad, I was just following a consistent pattern (mini cargo cult for this PR).